tracer/src/Datadog.Trace/Debugger/Expressions/ProbeExpressionParser.Collection.cs

```diff
@@ -149,7 +149,7 @@ private Expression CallGetItem(Expression source, Type assignableFrom, Expressio
         return Expression.Convert(getItemCall, convertToType);
     }
 
-    private Expression Count(JsonTextReader reader, List<ParameterExpression> parameters, ParameterExpression itParameter)
+    private Expression Length(JsonTextReader reader, List<ParameterExpression> parameters, ParameterExpression itParameter)
```

Comment:
Maybe instead of renaming `Count` -> `Length`, treat both the same?
This is a breaking change - if a customer is relying on `Count`, it won't work anymore upon library upgrade...

Reply:
Fixed in 3bcfaff0516e5c15d9b832a9e3b59d68fcaff851